CHANGELOG.md

```diff
@@ -5,6 +5,18 @@ This change log file is based on best practices from [Keep a Changelog](http://k
 This project adheres to [Semantic Versioning](http://semver.org/). Breaking changes result in a different MAJOR version. UI changes that might break customizations on top of the SDK will be treated as breaking changes too.
 This project adheres to the Node [default version scheme](https://docs.npmjs.com/misc/semver).
 
+## [2.3.0] - 16-04-2018
```

Comment:
I noticed that we never used to date releases. Anyhow i think this is good - we should date releases going forward like we already do on Android